exchange/utils_test.go

```diff
@@ -1045,6 +1046,8 @@ func TestCleanOpenRTBRequestsLMT(t *testing.T) {
 }
 
 func TestCleanOpenRTBRequestsGDPR(t *testing.T) {
+	TCF1Consent := "BONV8oqONXwgmADACHENAO7pqzAAppY"
```

Comment:
Nitpick: Method scoped variables should start with a lower case letter. tcf1Consent, tcf2Consent.